File: server/EAccountingServer.Application/Models/Dtos/Companies/CompanyListDto.cs
namespace EAccountingServer.Application.Models.Dtos.Companies
{
    public class CompanyListDto
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string FullAddress { get; set; }
        public string TaxDepartment { get; set; }
        public string TaxNumber { get; set; }
        //public Database Database { get; set; } = new(string.Empty, string.Empty, string.Empty, string.Empty);
    }
}

File: client/src/models/Companies/CompanyListDto.ts
import type { DatabaseListDto } from "../Databases/DatabaseListDto"

export class CompanyListDto{
    id: string = "";
    name: string = "";
    fullAddress: string = "";
    isDeleted: boolean = false;
    taxDepartment: string = "";
    taxNumber: string = "";
    database: DatabaseListDto | null= null
}